qiskit/transpiler/passmanager.py

```diff
@@ -72,40 +60,12 @@ def callback_func(**kwargs):
                     PassManager(callback=callback_func)
 
         """
-        self.callback = callback
-        # the pass manager's schedule of passes, including any control-flow.
-        # Populated via PassManager.append().
-        self.working_list = []
-
-        # global property set is the context of the circuit held by the pass manager
-        # as it runs through its scheduled passes. Analysis passes may update the property_set,
-        # but transformation passes have read-only access (via the fenced_property_set).
-        self.property_set = PropertySet()
-        self.fenced_property_set = FencedPropertySet(self.property_set)
-
-        # passes already run that have not been invalidated
-        self.valid_passes = set()
-
-        # pass manager's overriding options for the passes it runs (for debugging)
-        self.passmanager_options = {'max_iteration': max_iteration}
-
-        self.count = 0
-
+        self.pass_sets = []
```

Comment:
can you add docstring on the difference between `pm.passes` and `pm.pass_sets`?
it's not really clear from the names.

Reply:
should `pass_sets` be private?

Reply:
> can you add docstring on the difference between pm.passes and pm.pass_sets?

Done in e23d53f6

> should `pass_sets` be private?

Done in dd62a6c9